main.js

```diff
@@ -0,0 +1,329 @@
+  })
+  .then(function(data){
+    return data.json();
+  })
+  .then(data => addTodayForecastData(data));
+}
+
+// converts data from API call into object - calls rendering function for current day forecast
+let addTodayForecastData = function(data) {
+  today = getNameOfDay(data);
+  todaysForecast = {
+    temperature: `${Math.floor(data.main.temp)}°` ,
+    city: data.name,
+    clouds: data.weather[0].main
+  };
+  renderTodaysForecast();
+}
+// creates html template string and inserts element into html document 
+let renderTodaysForecast = function() {
+    document.querySelector('.today-forecast').replaceChildren();
+    let template = 
+    `<div id="today-forecast-data">
+        <h2 class="today-temp">${todaysForecast.temperature}</h2>
+        <h3 class="city">${todaysForecast.city}</h3>
+        <h4 class="cloud-cover">${todaysForecast.clouds}</h4>
+      </div>
+      <div id="big-weather-icon" class="mb-5">${selectBigWeatherIcon(todaysForecast.clouds)}</div>`;
+
+    let htmlObject = document.createElement('div');
+    htmlObject.className = 'row margin="15px"';
+    htmlObject.innerHTML = template;
+
+    document.querySelector('.today-forecast').insertAdjacentElement('beforeend', htmlObject);
+}
+// fetch to API for five day weather forecast
+let fetchFiveDayForecastData = function(searchInput) {
+  const fiveDayForecastURL = `https://api.openweathermap.org/data/2.5/forecast?q=${searchInput}&appid=${apiKey}&units=imperial`;
+
+  fetch(fiveDayForecastURL, {
+    method: 'GET',
+    datatype: 'json'
+  })
+  .then(data => data.json())
+  .then(data => addForecast(data)); 
+}
+
+/**
+ * 
+ * @param {JSON} data - converts data from API call into object
+ * calls function to render forecast for next five days
+ */
+let addForecast = function(data) {
+  // array to hold data from API call - converts data to array of 5 or 6 object items depending on time of day the API call is made
+  let fiveDayForecastArr = [];
+  fiveDayForecastsDOM.length = 0;
+
+  fiveDayForecastArr = getFiveDayForecastDayObjArr(data);
+  if(fiveDayForecastArr.length = 5){
+    for(let i = 0; i < fiveDayForecastArr.length; i++){
+      var forecastDayData = fiveDayForecastArr[i];
+      var forecast = {
+        city: forecastDayData.city,
+        dayOfWeek: forecastDayData.dayOfWeek,
+        temp: `${forecastDayData.temp}°`,
+        clouds: forecastDayData.clouds
+      }
+      fiveDayForecastsDOM.push(forecast);
+    }
+  } else if (fiveDayForecastArr.length = 6){
+    for(let i = 1; i < fiveDayForecastArr.length; i ++){
+      var forecastDayData = fiveDayForecastArr[i];
+      var forecast = {
+        city: forecastDayData.city,
+        dayOfWeek: forecastDayData.dayOfWeek, 
+        temp: `${forecastDayData.temp}°`,
+        clouds: forecastDayData.clouds
+      }
+      fiveDayForecastsDOM.push(forecast);
+    }
+  }
+  renderFiveDayForecast();
+}
+
+/**
+ * function creates HTML DOM template with values iterated from global variable

```

Comment:
nice job on comments! you could add @return void for functions that are just executing and not returning a data type